game-core/src/main/java/games/strategy/engine/chat/ChatController.java

```diff
@@ -20,6 +20,9 @@
 import games.strategy.net.Messengers;
 import games.strategy.util.Tuple;
 
+/**
+ * Default implementation of {@link IChatController}.
```

Comment:
This being an implementation of `IChatController` is made obvious by the code `implements IChatController`. 

When do we use this chat controller, what is it geared for, how is this implementation different from others? Should there just be a `@See` to reference the comment on `IChatController`, document it's role there, and then in the headless chat controller add a specific comment that we had to hack out chat.

Reply:
`ChatController` is the sole implementation of `IChatController`.  I think the `@link` reference serves the same role as a `@see` reference in this case.